recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4d5275a58da2a82fb550acbf496ff86e76d677e06b781baeefbe05a4cd47ab50
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - python
+    - pyee
+    - typing_extensions
```

Comment:
looking into source code
https://github.com/jonghwanhyeon/python-ffmpeg/blob/98707528ba424e414c5f3f9fe7dda9fbfed2018c/ffmpeg/ffmpeg.py#L23-L30

, it needs ffmpeg binary, does ffmpeg should not be added to run dependencies ?
Otherwise we will endup with non functional bindings ?

Reply:
it does make sense! Thanks! `ffmpeg` is not there for `s390x`, so I will skip it